CIRG-PROMIS-GLOBAL.json

```diff
@@ -2,6 +2,7 @@
   "resourceType": "Questionnaire",
   "id": "CIRG-PROMIS-GLOBAL",
   "title": "PROMIS short form - global - version 1.2",
```

Comment:
@mcjustin the `name` element is missing for this questionnaire.  When the frontend attempts to query using the name of each questionnaire , this is returned null.  Can that be added? As you have done with other questionnaires.

Reply:
I thought we were treating `id` as the canonical, where does the frontend need to query for `name`? There's overlap with `title` too, which is required by FHIR.

Reply:
Sorry, I am trying to figure out the best way  for the frontend to query for multiple questionnaires  all at once, I cannot use `_id`, it seems that FHIR doesn't allow it - at least it doesn't work as FHIR server doesn't return any result if I tried `_id=CIRG-PHQ9,CIRG=GAD7` It allows querying for multiple by name though. See [here](https://fhir.acc.dev.cosri.cirg.washington.edu/fhir/Questionnaire?name%3Acontains=peg%2Cbody+diagram%2Cphq9%2Cgad7%2C)  For context, in COSRI, [this](https://github.com/uwcirg/cosri-pain-management-summary/blob/ba68e72caaba9d0f139091486bb6285c27b2e46a/src/utils/executeELM.js#L333) is what I have done to query for questionnaires all at once. I could try query for each questionnaire individually by ID, I suppose, but I just need to re-structure the code a bit there.

Reply:
This query works for me, but perhaps I'm missing exactly what you need here:
```
https://fhir.acc.dev.cosri.cirg.washington.edu/fhir/Questionnaire?_id=CIRG-PHQ9,CIRG-GAD7
```

Reply:
sorry, @mcjustin, for not following up here.  But I did try it and it worked. Thank you for the suggestion.  Not sure why I didn't think to query this way.